### ISSUE TITLE
Why do you need containerHeight?

### ISSUE THREAD

**Issue by giantelk** [seatgeek/react-infinite#62]
Hi,

Why do you need to know/specify `containerHeight`? I was playing with the `Semantic UI` infinite scroll and it detects the bottom of a DOM element i.e. a `<div>` automatically without specifying the height of the `<div>` container. 

Here's a JSfiddle with Semantic UI's visibility for infinite scroll:
https://jsfiddle.net/4p6d7x86/13/

**Comment by garetht** [comment]
The code to do this simply has not yet been written.

**Comment by oliviertassinari** [comment]
:+1:

**Comment by yordis** [comment]
:+1:

**Comment by quarryman** [comment]
So is it going to be implemented right away?

**Comment by josercruz01** [comment]
+1

**Comment by garetht** [comment]
No, it is not going to be implemented right away, but if you have a pull request I will be happy to look at it. I imagine one of these days I will be given some time at work to look at this.

**Comment by carera** [comment]
+1

**Comment by zabojad** [comment]
+1

**Comment by long1110** [comment]
+1

**Comment by blazedd** [comment]
Is there any update on this issue?

**Comment by yordis** [comment]
@blazedd read the comment of @garetht

**Comment by geminiyellow** [comment]
but... now is 2017, Oct. one year later. @garetht do you give up this repo?
